gnuboardProject/frontend/src/components/Auth/Register.tsx

```
@@ -60,33 +226,211 @@ const Register: React.FC = () => {
 
+  const handleImgChange = (e: React.ChangeEvent<HTMLInputElement>) => {
+    if (e.target.files && e.target.files[0]) {
+      setProfileImg(e.target.files[0]);
+      setProfilePreview(URL.createObjectURL(e.target.files[0]));
+    }
+  };
+
+  const handleRemoveImg = (e: React.MouseEvent) => {
+    e.preventDefault();
+    setProfileImg(null);
+    setProfilePreview('');
```

Biome (1.9.4)

[error] Change to an optional chain.

Unsafe fix: Change to an optional chain.


(lint/complexity/useOptionalChain)